src/components/Header/header.css

```diff
@@ -0,0 +1,16 @@
+.burger-return-icon {
+    margin: 20px;
+    clip-path: circle(35% at center);
+    border: 1px solid transparent;
+    padding: 0.6em 1.2em;
+    font-family: inherit;
```

Comment:
what for you reset font here? we don't use it in buttons

Reply:
fixed